.env

```
@@ -10,4 +10,5 @@ DEV_FEEDBACK_API_URL=
 SENTRY_DSN=
 ENABLE_EXPERIMENTAL_TELEMETRY=false
 EXPERIMENTAL_TELEMETRY_ENDPOINT_URL=
-EXPERIMENTAL_TELEMETRY_TOKEN=
\ No newline at end of file
+EXPERIMENTAL_TELEMETRY_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPERIMENTAL_TELEMETRY_TOKEN key should go before the NEARBY_STATIONS_LIMIT key

(UnorderedKey)